src/main/resources/assets/advancedrocketry/recipes/satellitebuilder.json

```diff
@@ -18,36 +18,31 @@
             "item": "advancedrocketry:ic",
             "data": 0
         },
-	"b":
+        "b":
         {
             "item": "libvulpes:structuremachine"
         },
-	"m":
+        "m":
         {
 	    "type": "forge:ore_dict",
             "ore": "blockMotor"
         },
-	"t":
+        "t":
         {
 	    "type": "forge:ore_dict",
             "ore": "plateTitanium"
         },
-	"d":
+        "d":
         {
 	    "item": "advancedrocketry:dataunit",
 	    "data": 0
         },
-	"a":
+        "a":
         {
 	    "item": "minecraft:anvil",
 	    "data": 0
         },
-	"s":
-        {
-	    "item": "advancedrocketry:sawblade",
-	    "data": 0
-        },
```

Comment:
This entry used the same key as the User Interface, meaning only this was part of the recipe. It seemed out of place, and this machine certainly seemed like it should have a User Interface in crafting, so I removed the saw blade. Let me know if this recipe should have included a saw blade.

Reply:
For this I suggest checking what is was back on the master or 1.7.10 branch, as that would be before this move to JSON recipes could be made.

Reply:
Turned out to be the sawblade in 1.7.10, not the user interface. Changed in the new PR!